src/app/issuer/components/badgeclass-edit-form/badgeclass-edit-form.component.ts

```diff
@@ -230,6 +230,7 @@ export class BadgeClassEditFormComponent extends BaseAuthenticatedRoutableCompon
 	//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 	// Tags
 	tags = new Set<string>();
+	tagArray: string[] = [];
```

Comment:
Why do we need the `tagArray` and the `tags` set? This just seems duplicate and it worked before (the redesign of the badge creation) with just the `tags` set

Reply:
I agree the duplicate does not make sense. But are you sure this worked at one point? I went back to checkout different commits in history and didnt find this to be working yet

Reply:
I did successfully test it some time before the redesign, yes. Maybe you also need to check out the old backend for it to properly work? But either way, I think it should be possible without the duplicate.

Reply:
I still did not manage to find a point in time where this just worked with the autocomplete. I at least removed the duplicate now so that it works with the ```set```. The solution is not ideal, because when adding a tag and removing it afterwards it still shows in the list, i added a ```TODO``` for that

Reply:
I don't understand what you mean with the TODO. In the preview removing tags seems to work fine

Reply:
Removing a tag works, but it then still shows up in the list of existing tags in the dropdown